service-api/pom.xml

```diff
@@ -156,7 +157,27 @@
             <version>2.2.19</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-openfeign</artifactId>
+            <version>4.1.3</version>
+        </dependency>
     </dependencies>
 
-
+    <dependencyManagement>
```

Comment:
Any reason for using `dependencyManagement` only for this dependency?

Reply:
My bad, i'll refactored this!